src/Bundle/DataTransformer/ChainDataTransformer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\DataTransformer;
+
+use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
+
+final class ChainDataTransformer implements ChainDataTransformerInterface
```

Comment:
That's a neat solution, I like it!. It is heavily inspired by ApiPlatform, isn't it? 

One thing, I've learned from Sylius integration is to keep the possibility to have more than one data transformer for objects. Thanks to that, you can add some common metadata to objects/commands like data from the configuration, attributes or logged in user without calling it from the inside the application. I called it the "CommandEnrichment" in Sylius and perhaps we could implement it right away. The main idea is to keep running datatransformers after the first strike. On the other hand, we may make distinct concept for it

Reply:
@lchrusciel Yes, all this PR is inspired by ApiPlatform implementation 🥇 
There's no ChainDataTransformer on ApiPlatform, but the data transformer system is the same as the ApiPlatform one.